extensions/elasticsearch/7.x/wazuh-template.json

```diff
@@ -1148,6 +1182,117 @@
               }
             }
           },
+          "ms-graph": {
+            "properties": {
+              "relationship": {
+                "type": "keyword"
+              },
+              "classification": {
+                "type": "keyword"
+              },
+              "detectionSource": {
+                "type": "keyword"
+              },
+              "determination": {
+                "type": "keyword"
+              },
+              "remediationStatus": {
+                "type": "keyword"
+              },
+              "roles": {
+                "type": "keyword"
+              },
+              "verdict": {
+                "type": "keyword"
+              },
+              "serviceSource": {
+                "type": "keyword"
+              },
+              "severity": {
+                "type": "keyword"
+              },
+              "actorDisplayName": {
+                "type": "keyword"
+              },
+              "alertWebUrl": {
+                "type": "keyword"
+              },
+              "assignedTo": {
+                "type": "keyword"
+              },
+              "category": {
+                "type": "keyword"
+              },
+              "comments": {
+                "type": "keyword"
+              },
+              "createdDateTime": {
+                "type": "date"
+              },
+              "description": {
+                "type": "text"
+              },
+              "detectorId": {
+                "type": "keyword"
+              },
+              "evidence": {
+                "type": "nested",
+                "properties": {
+                  "_comment": {
+                    "type": "keyword"
+                  }
+                }
+              },
+              "firstActivityDateTime": {
+                "type": "date"
+              },
+              "id": {
+                "type": "keyword"
+              },
+              "incidentId": {
+                "type": "long"

```

Comment:
**suggestion**: maybe it should be a `keyword` too. I understand that this field is an identifier and should not be used as a number that can be compared with greater or lower values.

Reply:
It makes sense, so I have applied the suggested change.